tests/p2p/test_double_spending.py

```diff
@@ -165,7 +170,7 @@ def test_double_spending_propagation(self):
         outputs = [WalletOutputInfo(address=address, value=value, timelock=None),
                    WalletOutputInfo(address=address, value=tx_total_value - 500, timelock=None)]
         self.clock.advance(1)
-        inputs = [WalletInputInfo(i.tx_id, i.index, b'') for i in tx1.inputs]
+        inputs = [WalletInputInfo(i.tx_id, i.index, Mock()) for i in tx1.inputs]
```

Comment:
Why `Mock()`? I guess `b''` should work fine.

Reply:
This is the `WalletInputInfo` definition:

```
class WalletInputInfo(NamedTuple):
    tx_id: bytes
    index: Any  # FIXME: actually `int`, rename this field because `namedtuple` has an `index` method
    private_key: EllipticCurvePrivateKey
```

The third property expects an `EllipticCurvePrivateKey` object, and not bytes, so this was incorrect.